packages/core/content-manager/server/services/utils/configuration/metadatas.js

```diff
@@ -105,6 +105,12 @@ async function syncMetadatas(configuration, schema) {
       _.set(acc, [key], updatedMeta);
     }
 
+    // update visible attr
```

Comment:
Curious to know what usecase we are addressing here 🤔 ?

Reply:
Investigated a bit, I don't see what purpose does this serve 🤔 . I will dig a bit more and if I don't find anything I will remove it in this PR and discuss it with Fernando later.